### PR TITLE
admin_manual/list_of_events: Document dns client cached field

### PR DESCRIPTION
**Pull request by cmouse**
JIRA: DOV-4696

### CODE REVIEW

File: source/admin_manual/list_of_events.rst [hunk 1]
```diff
@@ -1650,6 +1650,9 @@ dns_request_finished
 +---------------------+------------------------------------------------------+
 | Field               | Description                                          |
 +=====================+======================================================+
+| cached              | Set to ``yes`` or ``no`` depending if it was         |
+|                     | a cached reply or not.                               |
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:1654]: versionadded needed also.

Comment (cmouse) [reply source/admin_manual/list_of_events.rst:1654]: should know what version to use though =)

Comment (sirainen) [reply source/admin_manual/list_of_events.rst:1654]: didn't we decide 3.0?..

File: source/admin_manual/list_of_events.rst [hunk 2]
```diff
@@ -1650,6 +1650,11 @@ dns_request_finished
...
+|                     |                                                      |
+|                     | .. versionadded:: 3.0                                |
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:1656]: Merge to master-3 branch until it's closer to being released?..

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/admin_manual/list_of_events.rst [hunk 3]
```diff
@@ -1770,6 +1770,11 @@ dns_request_finished
...
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:1776]: v2.4.0;v3.0.0

**Review by sirainen — DISMISSED** [review]

**Review by mrannanj — APPROVED** [review]

**Pull request merged**